tasks/e2e.sh

```diff
@@ -71,11 +71,10 @@ npm install
 
 # Test local build command
 npm run build
-# Check for expected output
+# Check for expected output (logo.svg is now inlined)
```

Comment:
There's two similar checks further down this file that still fail the test in Travis. Feel free to remove them too. Also the existing `# Check for expected output` comment is probably enough since nobody will remember `logo.svg` used to be not inlined. 😉